sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -18,23 +18,19 @@ export class SaveApplicationDto {
   /**
    * Application dynamic data.
    */
-
-  @IsNotEmpty()
+  @IsObject()
   data: any;
```

Comment:
Can we use the interface created for Application here?

Reply:
Yes, we can try and see if Nestjs will keep all dynamic properties. I believe that yes because it is an interface.
When I did it, it generated too many changes that would impact too many files outside this PR so I reverted it.